CHANGELOG.md

```diff
@@ -406,7 +406,7 @@ All notable changes to this project will be documented in this file.
   compatibility.
 * `cargo clippy --help` is more helping (and less helpful :smile:)
 * Rustup to *rustc 1.14.0-nightly (5665bdf3e 2016-11-02)*
-* New lints: [`if_let_redundant_pattern_matching`], [`partialeq_ne_impl`]
+* New lints: [`redundant_pattern_matching`], [`partialeq_ne_impl`]
```

Comment:
This should not be changed because it's the changelog of `0.0.97 — 2016-11-03`. Will update!